src/features/linkplugin/nodes/AutoLinkNodeModified.tsx

```diff
@@ -70,7 +71,7 @@ export function $createAutoLinkNode({
 }: {
   attributes?: LinkAttributes;
 }): AutoLinkNode {
-  return $applyNodeReplacement(new AutoLinkNode({ attributes: attributes }));
+  return $applyNodeReplacement(new AutoLinkNode({ attributes }));
```

Comment:
Is there by any chance a rule which does this the other way around: so enforces `attributes: attributes` instead of just `attributes`? I feel like it would add more clarity. What do you think?